frontend/src/components/UI/Form/ProjectOfficerReactSelect.jsx

```diff
@@ -0,0 +1,113 @@
+import { useEffect, useState } from "react";
+import { useGetUsersQuery } from "../../../api/opsAPI";
+import Select from "react-select";
```

Comment:
is this the new ComboBox?

Reply:
if so, should we remove the non-react-select `<ProjectSelect />` and `<ProjectOfficerSelect />`?

Reply:
Yes that is the proposed new combobox.  If we are happy with `react-select` we should replace the other implementations, e.g. `<ProjectSelect />` and `<ProjectOfficerSelect />` with it.  Our current implementation has a number of bugs and UX issues and wasn't working in the filter which led me to pick this alternative.  `react-select` should be accessible and seems to be working well.

Reply:
sounds good @johndeange. Sounds like we can use `react-select` to replace ComboBox across the app. May be a tech-debt item user story.